lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -284,7 +288,11 @@ def create(args)
           )
 
           unless bool
-            raise SystemCallError.new("LogonUserW", FFI.errno)
+            if FFI.errno == ERROR_LOGON_TYPE_NOT_GRANTED
+              raise SystemCallError.new("LogonUserW (You must hold `Log on as a service` and `Log on as a batch job` permissions.)", FFI.errno)
```

Comment:
Since we are using `LOGON32_LOGON_BATCH`, only "Log on as a batch job" is needed not "Log on as a service"